index.css

```diff
@@ -0,0 +1,339 @@
+/* 
+html, body, header, nav, h1, a,
+ul, ol, li, strong, main, button, i,
+section, img, div, h2, p, form,
+fieldset, label, input, textarea,
+span, article, footer, time, small {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  outline: 0;
+  font: inherit;
+  color: inherit;
+  text-align: inherit;
+  text-decoration: inherit;
+  vertical-align: inherit;
+  box-sizing: inherit;
+  background: transparent;
+}
+
+ul, ol {
+  list-style: none;
+}
+
+img {
+  display: block;
+  width: 100%;
+  height: auto;
+}
+
+input[type="password"],
+input[type="email"],
+input[type="text"],
+input[type="submit"],
+textarea,
+button {
+  -webkit-appearance: none;
+  -moz-appearance: none;
+}
+
+button,
+input[type="submit"] {
+  cursor: pointer;
+} */
+
+html, body{
+    margin: 0;
+    background-color: lightslategray;
+    height: 100%;
+    font-family: Arial, Helvetica, sans-serif;
+}
+#main{
+background-color:rgb(14, 151, 241) ;
+height: 100%;
+background-image: url("https://github.com/DejaLFlynn/Pursuit-Core-Web-Twitter-CSS-Assignment/blob/master/images/birdbackground.png?raw=true");
+background-size: cover;
+background-position: center;
+background-repeat:no-repeat;
+overflow: hidden;
+display: grid;
+grid-template-columns: repeat(2, 50%);
+
+
+
+}
+#left-side #right-side{
+    height: 100%;
+}
+#right-side{
+    background-color: white;
+    display: grid;
+
+    justify-content: center ;
+    height: 200%;
+    display: inline-block;
+    overflow: hidden;
+   
+    
+}
+
+#twitter{
+    display:grid;
+
+    /* justify-content: center; */
```

Comment:
you can add `justify-content: space-between` to give these divs some space around them